### DISCUSSION TITLE
Is there any Discord Server for gluesql?

### DISCUSSION THREAD

**Discussion by Raj2032** [gluesql/gluesql#535] [Q&A] [answered]
Is there any Discord Server for gluesql? If not then shall I start one?

**Comment by panarch** [comment, marked as answer]
Yes here it is - https://discord.gg/C6TDEgzDzY
There is also a link badge in README.md but it is quite small to recognize.
![image](https://user-images.githubusercontent.com/2025065/164910324-f9327bb8-85d7-4259-9402-1fc83a9c965a.png)
